app/dashboard/templates/ftux/onboard.html

```diff
@@ -72,11 +74,17 @@
       steps.push("{{step}}");
     {% endfor %};
   </script>
+  {% compress js %}
   <script src="{% static "v2/js/pages/onboard.js" %}"></script>
+  {% endcompress %}
   <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBaJ6gEXMqjw0Y7d5Ps9VvelzOOvfV6BvQ&libraries=places&callback=initPlacecomplete"></script>
+  {% compress js %}
   <script src="{% static "v2/js/jobs.js" %}"></script>
   <script src="{% static "v2/js/avatar_builder.js" %}" defer></script>
+  {% endcompress %}
```

Comment:
would it be possible to move any js files that are not compressable (because they are hosted elsewhere, or they are dynamically generated like `tokens_dynamic`) to below the compress block and then only have ONE compress block?

Reply:
this applies in more than just this file ..

Reply:
Sure, I'm working on that !